site/en/blog/compression-streams-api/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Compression and decompression in the browser with the Compression Streams API
+subhead: >
+  Write smaller web apps that don't need to ship their own compression or decompression library
+date: 2022-08-26
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/koW7cNlWa6BUlJsQR4wI.jpg
+alt: Clamp to symbolize how data gets compressed.
+author: thomassteiner
+tags:
+  - capabilities
+---
+
+The [Compression Streams API](https://developer.mozilla.org/docs/Web/API/Compression_Streams_API)
+provides a JavaScript API for compressing and decompressing streams of data using the gzip or
+deflate (or deflate-raw) formats.
+
+Built in compression means that JavaScript applications will not need to include a compression
+library, which makes the download size of the application smaller. After Chrome, now Safari, too,
```

Comment:
```suggestion
library, making the download size of the application smaller. Chrome and Safari now
```

Reply:
ah is it just in TP? If so we can't say it's supported in Safari as we don't know when. Is it in the 16 beta? If not I'd say it's coming soon to Safari.

Reply:
I'll just say STP instead of Safari then.